lib/easybill-rest-client/generic_api.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+module EasybillRestClient
+  class GenericApi
```

Comment:
Now I think, as we don't actually instantiate the class, but only use it as a factory, it could also be a `module` instead of a `class`.